alerting/src/utils/analysis-utils.ts

```diff
@@ -1,6 +1,6 @@
 import fetch from 'node-fetch';
 import { get, isNil } from 'lodash';
-import bbox from '@turf/bbox';
+import { bbox } from '@turf/turf';
```

Comment:
it is recommended to use subpackages. Turf library itself might increase bundle size drastically, and currently prism does not make use of a lot of turf functions

Reply:
👍